compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -651,6 +651,10 @@ def PadDynamicAllocPass :
 def MathTransformPass :
     Pass<"iree-codegen-math-transform", ""> {
   let summary = "Apply math ops transformations: approximations, rewrites to other math ops, operand casts.";
+  let dependentDialects = [
+    "::mlir::scf::SCFDialect",
+    "::mlir::arith::ArithDialect"
```

Comment:
(this file has a few consistency issues but predominantly does not lead with `::`)

```suggestion
    "mlir::scf::SCFDialect",
    "mlir::arith::ArithDialect"
```